alerts/proxy_drop_executable.conf

```diff
@@ -0,0 +1,2 @@
+[options]
+destinatationfilter = (repomd.xml$)
```

Comment:
This is to work around an issue where by CentOS will ignore provided repositories.  We will not alert on those cases to try and get a better signal to noise.  There is not much concern around an attacker knowing this, as the traffic is being dropped anyways.